libs/wlgen/wlgen/rta.py

```diff
@@ -101,6 +101,39 @@ def __init__(self,
         # Setup RTA callbacks
         self.setCallback('postrun', self.__postrun)
 
+    @staticmethod
+    def __setGovernorsPerformance(target):
```

Comment:
Double leading underscore has a special meaning in Python and must not be used unless there is a good reason. This definition will define: RTA._RTA__setGovernorsPerformance, which is unlikely to be what you expect, and will prevent subclasses from overriding it in a sane way. Internal methods can use a single underscore to mark them as not exposed as a user API.

Details on name mangling here: https://docs.python.org/2/tutorial/classes.html#private-variables-and-class-local-references